Automattic/wp-calypso
# AB Test: Wrapping up Checkout UX improvements

#### Changes proposed in this Pull Request

* Removing `CreditCardFormFieldsUX` component and implementing the proposed changes to  `CreditCardFormFields` as new control.

#### Testing instructions

Visit the URL:  https://calypso.live/?branch=update/checkout-implementing-ux-improvements

1. Go to a testing site or create a new one.

2. Purchase or upgrade to a plan to trigger Checkout. Also, you can directly go to a Checkout URL such as http://calypso.live/checkout/[YOUR_SITE]/blogger as seen [here](https://github.com/Automattic/wp-calypso/tree/master/client/my-sites/checkout)

4. Confirm you're seeing the Checkout form with UX improvements. Pay attention to placeholders and gray text explanations on `"Cardholder Name"` and `"Security Code"`.

![image](https://user-images.githubusercontent.com/538790/57229023-6dc8a180-7015-11e9-82c4-cd5041d8ddc1.png)

5. Go to /me -> Manage Purchases -> Billing History -> Add Credit Card ([link](https://calypso.live/me/purchases/billing)). Make sure the Credit Card form is the UX-improved one.
